.github/workflows/upload.yaml

```diff
@@ -0,0 +1,77 @@
+name: Upload to website
+
+on:
+  pull_request:
+    branches:
+      - main
+
+  workflow_dispatch:
+
+jobs:
+  Upload-staging:
+    if: ${{ github.event_name == 'pull_request' && github.event.action == 'closed' && github.event.pull_request.merged == true }}
```

Comment:
- If we want to keep the workflow_dispatch, I think some of these events won't resolve.
- We should be able to refactor this into a composite action and eliminate the repeated steps.
- I don't actually know the purpose of staging versus dev environment.  We may need to draw a picture to understand when assets are pushed. 
- Should we use some of the features available in [repository environments](https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment)?

Reply:
The workflow_dispatch might help in deciding where we actually want some changes deployed to based on environments already setup.

Reply:
once this is in https://github.com/Opentrons/opentrons-python-packages/blob/main/.github/workflows/build-packages.yaml we'll be able to take advantage of that branch's workflow dispatch rules.